### DISCUSSION TITLE
How do I include photos from my actual SD card?

### DISCUSSION THREAD

**Discussion by pegelf** [deckerst/aves#1361] [Q&A]
Hi. I really liked to use this Gallery App on my old phone, because I have 300GB of photos in different folders and this App had the best performance on opening them.

On my new phone, I noticed that the stock Android ROM didn't support SD Card encryption, so as a workaround, I created a [script](https://github.com/pegelf/Android-LUKS-mount) to mount a LUKS-encrypted SD Card in the "SD" folder of the internal storage. Inside of this Folder, I have a folder called "Bilder" which contains my Albums.

Of course, file managers can open that folder but your Gallery seems to follow specific folder rules which I haven't fully understood yet. Is it possible to create "Albums" or "Pictures" folders inside the "SD" folder? Or do ADB commands exist to define which folders are media folders?

**Comment by deckerst** [comment]
The app shows files recognized as media at the OS level. In other words, the media files need to be registered (or discovered) by the OS for Aves to be able to show them. 

Usually this is not a problem because files are moved to SD cards via MTP, or created on it by apps which take care of media registration. 

With your approach, files are moved, but they are not registered as media, so you will need to wait for the OS to discover them. Discovery can take a long time. Also, the OS never scans transient mounts (like OTG) for media registration. 

With regards to album and media creation on your SD card within Aves, the SD card partition should be selectable in the album creation dialog:
![Screenshot_20241227-230349~2](https://github.com/user-attachments/assets/b1f1e3f0-8678-43eb-80c6-f60e7b6fdeea)

**Comment by pegelf** [comment]
Thank you for your quick reply.

After one week my photos are still not recognited as media and not showed in Aves. Also I saw in other discussions that including folders manually isn't planned currently.

So I guess I can't use Aves for opening photos from my SD card?

**Comment by deckerst** [comment]
> So I guess I can't use Aves for opening photos from my SD card?

Not the way you do it. But if you were using the SD card the usual way you could.
